src/log/io_backend.cr

```diff
@@ -4,7 +4,7 @@ class Log::IOBackend < Log::Backend
   property progname : String
   property formatter : Formatter? = nil
```

Comment:
```suggestion
  property formatter : Formatter?
```
since it's set to nil in the initialize ?

Reply:
Yep!